src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -282,6 +282,30 @@ paths:
                 $ref: "#/components/schemas/TransactionCommittedDetailsResponse"
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
+  "/transaction/committed-outcome":
+    post:
+      operationId: TransactionCommittedOutcome
+      summary: Get Committed Transaction Outcome
+      description: |
+        Returns the committed outcome of the transaction for a given transaction identifier.
+        Transaction identifiers which don't correspond to a committed transaction will return a `TransactionNotFoundError`.
```

Comment:
out of curiosity will it support both commited success and commited failures? hmm what would be outcome of committed failure...

Reply:
1. I've added committed tx info (lightweight object) to convey successful/faulted info.
2. For failed tx you might still be a subject to various fees.